components/massmint/OnBoarding.vue

```diff
@@ -0,0 +1,250 @@
+        :class="`slide-${currentSlide}`"
+        @touchstart="onTouchStart"
+        @touchmove="onTouchMove"
+        @touchend="onTouchEnd">
+        <div
+          v-for="(card, index) in cards"
+          :key="index"
+          class="carousel-card p-5 mobile-padding"
+          :class="{ 'not-active': index !== currentSlide }">
+          <div class="card__content">
+            <p
+              class="title is-size-2-desktop is-size-3-tablet is-size-5-mobile is-capitalized">
+              {{ card.title }}
+            </p>
+            <div class="content is-size-4-tablet is-size-5-mobile">
+              <VueMarkdown :source="card.content" />
+            </div>
+          </div>
+        </div>
+      </div>
+      <Transition name="fade">
+        <div
+          v-if="currentSlide > 0"
+          class="arrow arrow-left"
+          @click="prevSlide" />
+      </Transition>
+      <Transition name="fade">
+        <div
+          v-if="currentSlide < numOfCards - 1"
+          class="arrow arrow-right"
+          @click="nextSlide" />
+      </Transition>
+    </div>
+
+    <div class="is-flex is-justify-content-center my-6">
+      <span
+        v-for="(card, index) in cards"
+        :key="index"
+        class="carousel-dot mx-2"
+        :class="{ 'is-active': index === currentSlide }"></span>
+    </div>
+    <div class="is-flex is-justify-content-center">
+      <NeoButton
+        :label="btn.label"
+        class="is-flex-grow-1 limit-width"
+        :variant="btn.variant"
+        @click.native="btn.onClick" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue'
+import { NeoButton, NeoButtonVariant } from '@kodadot1/brick'
+import VueMarkdown from 'vue-markdown-render'
+import { useMassmintsStore } from '@/stores/massmint'
+
+const { $consola } = useNuxtApp()
+const router = useRouter()
+const { urlPrefix } = usePrefix()
+const { $i18n } = useNuxtApp()
+
+const numOfCards = 3
+const massMintStore = useMassmintsStore()
+const currentSlide = ref(0)
+const startX = ref(0)
+const startY = ref(0)
+const isSwiping = ref(false)
+
+const cards = computed(() => {
+  const indices = Array.from({ length: numOfCards }, (_, i) => i + 1)
+  return indices.map((i: number) => ({
+    title: $i18n.t(`massmint.onboarding.cards.${i}.title`),
+    content: $i18n.t(`massmint.onboarding.cards.${i}.content`),
+  }))
+})
+
+const nextSlide = () => {
+  if (currentSlide.value < numOfCards - 1) {
+    currentSlide.value++
+  }
+}
+const prevSlide = () => {
+  if (currentSlide.value > 0) {
+    currentSlide.value--
+  }
+}
+
+const toMassMint = () => {
+  massMintStore.setVisitedOnboarding(true)
+  router
+    .replace({
+      path: `/${urlPrefix.value}/massmint`,
+    })
+    .catch($consola.warn)
+}
+
+const btn = computed(() =>
+  currentSlide.value === 2
+    ? {
+        label: $i18n.t('massmint.onboarding.done'),
+        variant: 'k-accent' as NeoButtonVariant,
```

Comment:
`as NeoButtonVariant`
Is it necessary to have this

Reply:
![image](https://user-images.githubusercontent.com/22791238/224063277-d889456e-df3b-4c76-85fa-18e6a94d3188.png)


```Type 'string' is not assignable to type 'NeoButtonVariant | undefined'```